main/src/ca/uwaterloo/flix/language/ast/Type.scala

```diff
@@ -325,6 +326,11 @@ object Type {
     */
   val Pure: Type = Type.Cst(TypeConstructor.Pure, SourceLocation.Unknown)
 
+  /**
+    * Represents the IO effect.
+    */
+  val IO: Type = Type.Cst(TypeConstructor.Effect(Symbol.mkEffectSym(Name.RootNS, Name.Ident(SourcePosition.Unknown, "IO", SourcePosition.Unknown))), SourceLocation.Unknown)
```

Comment:
I was thinking of hardcoding the source position of IO. (i.e. we know where it is in Prelude), but I did not get to actually try it. I think such an approach could be generally useful. Perhaps it will not be feasible until we replace SourcePosition by something else when we get to the new Parser.

Reply:
see later comment about the symbol.

Reply:
I was thinking that IO might be considered a built-in type rather than part of the standard library, since it is so integral to the language.